relayer/src/chain/cosmos.rs

```diff
@@ -1385,11 +1385,9 @@ async fn abci_query(
         return Err(Kind::EmptyResponseProof.into());
     }
 
-    let raw_proof_ops = response.proof;
-
     let response = QueryResponse {
         value: response.value,
-        proof: convert_tm_to_ics_merkle_proof(raw_proof_ops).unwrap(), // FIXME
+        proof: convert_tm_to_ics_merkle_proof(response.proof).unwrap(), // FIXME
```

Comment:
Can we address the `unwrap` here while we're at it?

Reply:
caf78587

Reply:
Sorry I should have been more clear, doing this required also addressing the `unwrap` in `convert_tm_to_ics_merkle_proof`. Fixed in 75610d9.